manage-images.md

```diff
@@ -0,0 +1,120 @@
+An image is the base for a container running in the Anbox Cloud. It contains all necessary components, like Anbox or the Android root file system. Each release of Anbox Cloud comes with an update image, which must be uploaded to the AMS service to make it available to be used inside any container.
+
+AMS can manage different types of images, which can be individually selected by applications. For that, each image must be named when it's initially uploaded to AMS. If the same image receives an update with the next release of Anbox Cloud, it does not need to be re-created but can simply be updated. This will cause all applications using the image to be updated and rebased on the new image version.
+
+Officially released images are available for each version of Anbox Cloud, either on the [Downloads](https://oem-share.canonical.com/partners/indore/share/docs/1.7/en/installation-downloads.html) page (for versions <= 1.7) or on an image server hosted by Canonical.
```

Comment:
Is it correct that the Downloads page is only for versions up to 1.7?

Reply:
@ru-fu that's correct. The delivery mechanism was more manual before, but the introduction of the image server changed things quite a bit